index.html

```diff
@@ -0,0 +1,236 @@
+            <h1 class="PrimaryText">Hey There. I'm faishar</h1>
+          </div>
+          <div class="SupportingText">
+            <p class="SupportingText">I'm a Software Developer</p>
+          </div>
+          <div class="discription">
+            <p class="discription">
+              I can help you build a product , feature or website Look through
+              some of my work and experience! If you like what you see and have
+              a project you need coded, don't hestiate to contact me.
+            </p>
+          </div>
+          <ul class="SocialMedia">
+            <li>
+              <a href="https://github.com/Robison-faike123">
+                <img src="images/Icon -GitHub.png" alt="-GitHub" />
+              </a>
+            </li>
+            <li>
+              <a href="https://www.linkedin.com/">
+                <img src="images/Icon - Linkedin.png" alt="Linkedin" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="images/Icon - Angelist.png" alt="Angelist" />
+              </a>
+            </li>
+            <li>
+              <a href="https://twitter.com">
+                <img src="images/Icon - Twitter.png" alt="Twitter" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="images/Icon - Medium.png" alt="Medium" />
+              </a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <!--this is for second section -->
+    <section class="Works">
+      <div class="RecentWorks">
+        <h1>My Recent Works</h1>
+      </div>
+      <div class="MultiPostStories">
+        <img
+          class="MyResentWorkImg1"
+          src="/images/MyRecentWorkImg1.png"
+          alt=""
+        />
+    
+        <h2>Multi-Post Stories</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <ul class="languageBoxes">
+          <li class="box">
+            <p>CSS</p>
+          </li>
+          <li class="box">
+            <p>HTML</p>
+          </li>
+          <li class="box">
+            <p>BOOTSTRAP</p>
+          </li>
+          <li class="box">
+            <p>RUBY</p>
+          </li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    
+      <div class="work1" id="w1">
+        <h1>Profesional Art Printing Data</h1>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="category">
+          <li class="language1">html</li>
+          <li class="language2">bootstrap</li>
+          <li class="language3">Ruby</li>
+        </ul>
+        <button>See Project</button>
```

Comment:
- [ ] When I try to hover over the button, I am not getting the desired button state outcome. Kindly ensure that you add the on hover to change the text color as well as the active when someone clicks on a button. Below is how I am seeing it right now.

[Screencast from 28-09-2022 10:29:51 ASUBUHI.webm](https://user-images.githubusercontent.com/86318284/192716262-68162587-f6db-46a4-90e7-cc2c38a36fd7.webm)

**These are the desired button states**

![image](https://user-images.githubusercontent.com/86318284/192716746-364bc3a1-031b-41b0-8b41-49d1ffe687d3.png)

**Kindly ensure you make this change in all the buttons in the website as I have realized the trend is the same there as well**

Reply:
hover is worked on and all button have hovers now, the thing on the instructions paper was to do something similar not exact as it is I may want to change the color which is attractive to me but you don't give chance to enjoy studying